### PR TITLE
New questions on html and css pages

### PR DESCRIPTION
**Pull request by devshittu**
I have added 5 new questions on the HTML page and 2 new questions on the CSS page. There are no required dependencies for this update,

Fixes # (issue): 
None

## Type of change: 

Please delete options that are not relevant.

- [x] New Question
- [ ] Revision of an existing question
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate) 

# Checklist:

- [*] My content follows the style guidelines of this project
- [*] I have performed a self-review of my own content

Pull requests should be thought of as a conversation. There will be some back and forth when trying to get code merged into this or any other project. With all but the simplest changes you can and should expect that the maintainers of the project will request changes to your code. Please be aware of that and check in after you open your PR in order to get your code merged in cleanly. 

Thanks!

### CODE REVIEW

**Comment by LantingHe** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。

File: src/questions/html-questions.md [hunk 1]
```diff
@@ -17,3 +17,8 @@ permalink: /questions/html-questions/index.html
 * Have you used different HTML templating languages before?
 * What is the difference between `canvas` and `svg`?
 * What are empty elements in HTML ?
+* What are the semantic HTML5 page structure elements?
+* What is an API for HTML5? Can you name some APIs in HTML5?
+* What is an iFrame? What are the risks of using it within your code?
+* What are the media element tags introduced by HTML5?
+* Describe how can a copyright symbol be added to the webpage using HTML5?
```
Comment (roblarsen) [inline src/questions/html-questions.md:24]: I would love to see a question about character encoding (if there's not one.) This is a 2 second answer.

File: src/questions/html-questions.md [hunk 2]
```diff
@@ -17,3 +17,8 @@ permalink: /questions/html-questions/index.html
 * Have you used different HTML templating languages before?
 * What is the difference between `canvas` and `svg`?
 * What are empty elements in HTML ?
+* What are the semantic HTML5 page structure elements?
+* What is an API for HTML5? Can you name some APIs in HTML5?
+* What is an iFrame? What are the risks of using it within your code?
+* What are the media element tags introduced by HTML5?
```
Comment (roblarsen) [inline src/questions/html-questions.md:23]: There are good questions about embedding media, not just listing the elements.

File: src/questions/css-questions.md [hunk 3]
```diff
@@ -39,4 +39,6 @@ permalink: /questions/css-questions/index.html
 * Can you explain the difference between px, em and rem as they relate to font sizing?
 * Can you give an example of a pseudo class? Can you provide an example use case for a pseudo class? 
 * What is the difference between a block level element and an inline element. Can you provide examples of each type of element?
-* What is the difference between CSS Grid and Flexbox? When would you use one over the other?
\ No newline at end of file
+* What is the difference between CSS Grid and Flexbox? When would you use one over the other?
```
Comment (roblarsen) [inline src/questions/css-questions.md:42]: This is a good discussion question

File: src/questions/css-questions.md [hunk 4]
```diff
@@ -39,4 +39,6 @@ permalink: /questions/css-questions/index.html
...
+* What are the Flexbox advantages compared to floats?
```
Comment (roblarsen) [inline src/questions/css-questions.md:43]: You could reword this to be something like "Compare flexbox with legacy layout techniques."

File: src/questions/css-questions.md [hunk 5]
```diff
@@ -39,4 +39,6 @@ permalink: /questions/css-questions/index.html
...
+* What properties does justify-content and align-items share in CSS Flexbox?
```
Comment (roblarsen) [inline src/questions/css-questions.md:44]: skip this one

File: src/questions/html-questions.md [hunk 6]
```diff
@@ -17,3 +17,8 @@ permalink: /questions/html-questions/index.html
 * Have you used different HTML templating languages before?
 * What is the difference between `canvas` and `svg`?
 * What are empty elements in HTML ?
+* What are the semantic HTML5 page structure elements?
+* What is an API for HTML5? Can you name some APIs in HTML5?
+* What is an iFrame? What are the risks of using it within your code?
```
[the remark this answers is not in the document]
Comment (roblarsen) [reply src/questions/html-questions.md:22]: ✅

File: src/questions/css-questions.md (hunk 3 above)
Comment (roblarsen) [reply src/questions/css-questions.md:42]: ✅

File: src/questions/css-questions.md (hunk 4 above)
Comment (roblarsen) [reply src/questions/css-questions.md:43]: ✅

**Review by roblarsen — CHANGES_REQUESTED** [review]
If you're still interested, reduce the PR to the questions with ✅ and then we can work one more round on the wording. Thanks!
